src/components/Shooting.js

```diff
@@ -0,0 +1,65 @@
+AFRAME.registerComponent('shooting-function', {
+    schema: {
+      velocity: {type: 'int', default: 15}
+    },
+    init: function () {
+      console.log("in init")
+    },
+    udpate: function () {
+      console.log("in the update")
+      console.log(this.el.object3D.position)
+    },
+    tick: function () {
+      // checks for when the ball is below y= -100, then returns it to the pool
+      let sceneEl =document.querySelector('#ball-pool');
+      // console.log(this.el.getAttribute('position').y )
+      if (this.el.getAttribute('position').y <-100) {
+        sceneEl.components.pool__projectile.returnEntity(this.el);
+        console.log("retured!")
+      }
+    },
+    remove: function () {},
+    pause: function () {
+      console.log("pause")
+    },
+    play: function () {
```

Comment:
Also can you change the name from "play" to "shoot" or "fire" to be more specific to this component's functionality

Reply:
Here are the docs on how  to call Component Member Methods https://aframe.io/docs/1.2.0/core/component.html#accessing-a-component%E2%80%99s-members-and-methods